src/layouts/SiteLaunchPad/SiteLaunchPad.tsx

```diff
@@ -0,0 +1,159 @@
+            If you are unsure, consult your ITD team on this. If your domain is
+            a .gov.sg domain, You can go to{" "}
+            <Link href="https://dnschecker.org/">DNS checker</Link> and check if
+            your A record starts with 35.
+          </Text>
+          <br />
+          <Text textStyle="body-2">
+            Isomer will not be held liable for any extended period of downtime
+            due to unknown cloudfront records. Downtime could happen even if
+            your site is not currently using cloudfront, but has used it
+            previously.
+          </Text>
+          <br />
+          <br />
+          <Checkbox
+            onChange={(e) => {
+              e.preventDefault()
+              setIsDisabled(!e.target.checked)
+            }}
+          >
+            <Text textStyle="body-1" color="black">
+              I understand the risks
+            </Text>
+          </Checkbox>
+        </ModalBody>
+
+        <ModalFooter>
+          <Button
+            mr={3}
+            onClick={() => setPageNumber(SITE_LAUNCH_PAGES.INFO_GATHERING)}
+            variant="link"
+          >
+            Cancel
+          </Button>
+          <Button
+            isDisabled={isDisabled}
+            type="submit"
+            onClick={() => setPageNumber(SITE_LAUNCH_PAGES.CHECKLIST)}
+          >
+            Continue
+          </Button>
+        </ModalFooter>
+      </ModalContent>
+    </Modal>
+  )
+}
+const pageInitialNumber = (
+  siteLaunchStatusProps: SiteLaunchStatusProps | undefined
+) => {
+  const hasUserAlreadyStartedChecklistTasks =
+    siteLaunchStatusProps?.siteLaunchStatus === "CHECKLIST_TASKS_PENDING"
+  if (hasUserAlreadyStartedChecklistTasks) return SITE_LAUNCH_PAGES.CHECKLIST
+  const isSiteLaunchInProgress =
+    siteLaunchStatusProps?.siteLaunchStatus === "LAUNCHING"
+  if (isSiteLaunchInProgress) return SITE_LAUNCH_PAGES.CHECKLIST
+  return SITE_LAUNCH_PAGES.DISCLAIMER
+}
+export const SiteLaunchPad = (): JSX.Element => {
+  const {
+    siteLaunchStatusProps,
+    setSiteLaunchStatusProps,
+  } = useSiteLaunchContext()
+
+  const [pageNumber, setPageNumber] = useState(
+    pageInitialNumber(siteLaunchStatusProps)
+  )
+
+  let title = <SiteLaunchDisclaimerTitle />
+  let body = <SiteLaunchDisclaimerBody setPageNumber={setPageNumber} />
+  switch (pageNumber) {
+    case SITE_LAUNCH_PAGES.DISCLAIMER:
+      title = <SiteLaunchDisclaimerTitle />
+      body = <SiteLaunchDisclaimerBody setPageNumber={setPageNumber} />
+      break
+    case SITE_LAUNCH_PAGES.INFO_GATHERING:
+    case SITE_LAUNCH_PAGES.RISK_ACCEPTANCE: // Risk acceptance modal overlay
+      title = <SiteLaunchInfoGatheringTitle />
+      body = <SiteLaunchInfoGatheringBody setPageNumber={setPageNumber} />
+      break
+
+    // todo case for site launch pages checklist
+    default:
+      title = <SiteLaunchDisclaimerTitle />
+      body = <SiteLaunchDisclaimerBody setPageNumber={setPageNumber} />
+  }
```

Comment:
hey uhm actually since got the default there shouldnt be reassignment here. I am ok to shift this out into a component, but that would still have the switch case statement. I have removed the initial assignemnt for now, let me know which one you prefer

Reply:
was thinking of shifting it out to another component (can be here also) then directly `switch + return` - downside is that it's not so obvious what's `title/body`. 

i think this is p minor so up 2 u